src/controllers/CheckYourAppealController.ts

```diff
@@ -46,6 +49,16 @@ export class CheckYourAppealController extends BaseAsyncHttpController {
             .chain(data => Maybe.fromNullable(data[APPEALS_KEY]))
             .extract() as Appeal;
 
+        const accessToken = req.session
+            .chain(_ => _.getValue<ISignInInfo>(SessionKey.SignInInfo))
+            .map(info => info[SignInInfoKeys.AccessToken])
+            .map(token => token?.access_token)
+            .extract() as string;
+
+        const companyNumber: string = appealsData.penaltyIdentifier.companyNumber;
+
+        await this.appealSubmissionService.submitAppeal(appealsData, companyNumber, accessToken);
```

Comment:
A `appealSubmissionService.submitAppeal(appeals` is looks very repetitive. In Poland we say "Masło maślane" which translated means "Buttery butter" :) There is no need to repeat submit and appeal verbs so many times.

Reply:
I was trying to think of a good name for this, how about: `appealStorageService.save()`? or any suggestions?